apps/core/src/hooks/useGetObject.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRpcClient } from '../api/RpcClientContext';
+import { normalizeSuiAddress, type SuiObjectDataOptions } from '@mysten/sui.js';
+import { useQuery } from '@tanstack/react-query';
+
+const defaultOptions: SuiObjectDataOptions = {
+    showType: true,
+    showContent: true,
+    showOwner: true,
+    showDisplay: true,
+};
+
+export function useGetObject(
+    objectId?: string | null,
+    options = defaultOptions
```

Comment:
This would need to be part of the cache key if we want to support this. But also am open to just not supporting variable display options.

Reply:
Ok, I will remove this input option